packages/textarea/src/index.ts

```diff
@@ -1,9 +1,33 @@
-import { FrameworkConfiguration, PLATFORM } from 'aurelia-framework';
+import { FrameworkConfiguration, PLATFORM, bindingMode, ValueAttributeObserver } from 'aurelia-framework';
+import { AureliaUX } from '@aurelia-ux/core';
 
 export { UxTextareaTheme } from './ux-textarea-theme';
 
 export function configure(config: FrameworkConfiguration) {
+  config.container.get(AureliaUX).registerUxElementConfig(uxTextAreaConfig);
   config.globalResources([
     PLATFORM.moduleName('@aurelia-ux/textarea/ux-textarea')
   ]);
 }
+
+const uxTextAreaConfig = {
```

Comment:
We need to be sure to extend this configuration to the [`@aurelia-ux/components`](https://github.com/aurelia/ux/tree/master/packages/components) package.